modules/core/src/v2/coins/trx.ts

```diff
@@ -15,11 +16,30 @@ import {
   SignTransactionOptions,
   VerifyAddressOptions,
   VerifyTransactionOptions,
+  TransactionExplanation,
 } from '../baseCoin';
 import * as utxoLib from 'bitgo-utxo-lib';
 import { BitGo } from '../../bitgo';
 import { NodeCallback } from '../types';
 
+const co = Bluebird.coroutine;
+
+export interface TronSignTransactionOptions extends SignTransactionOptions {
+  txPrebuild: TransactionPrebuild;
+  prv: string;
+}
+
+export interface TransactionPrebuild {
```

Comment:
This should `extend BaseTransactionPrebuild` from baseCoin so we have some common interface between coins